examples/tooltip-on-hover.js

```diff
@@ -0,0 +1,75 @@
+import Fill from '../src/ol/style/Fill.js';
+import GeoJSON from '../src/ol/format/GeoJSON.js';
+import Map from '../src/ol/Map.js';
+import Style from '../src/ol/style/Style.js';
+import VectorLayer from '../src/ol/layer/Vector.js';
+import VectorSource from '../src/ol/source/Vector.js';
+import View from '../src/ol/View.js';
+
+const style = new Style({
+  fill: new Fill({
+    color: '#eeeeee',
+  }),
+});
```

Comment:
This will need to go away too.  And the `import`s for `Fill` and `Style` above.

Reply:
Ah sorry for that. Done.